com.tencent.WeChat.yaml

```diff
@@ -0,0 +1,105 @@
+id: com.tencent.WeChat
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: wechat
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  # System tray icon
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
+  # Files
+  - --filesystem=xdg-download
+  - --persist=.xwechat
+  - --persist=xwechat_files
+  # Hidpi scale
+  - --env=QT_AUTO_SCREEN_SCALE_FACTOR=1
+  # Fix `Could not find the Qt platform plugin "wayland"`
+  - --unset-env=QT_QPA_PLATFORM
```

Comment:
```suggestion
```

This is a problem with your setup, Qt does not use wayland by default.

Reply:
This app only has xcb plugin inside. I think it would be safe to explicitly add QT_QPA_PLATFORM=xcb here.